src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs

```diff
@@ -99,6 +99,31 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
                                 PrivateAssets = "All"
                             }), noFrameworkPackageReferenceItemGroup, mergeExisting: false);
                     break;
+                case ProjectType.Test:
+                    _transformApplicator.Execute(
+                        PackageDependencyInfoTransform.Transform(
+                            new PackageDependencyInfo
+                            {
+                                Name = PackageConstants.TestSdkPackageName,
+                                Version = PackageConstants.TestSdkPackageVersion
+                            }), noFrameworkPackageReferenceItemGroup, mergeExisting: false);
+
+                    _transformApplicator.Execute(
+                        PackageDependencyInfoTransform.Transform(
+                            new PackageDependencyInfo
+                            {
+                                Name = PackageConstants.XUnitPackageName,
+                                Version = PackageConstants.XUnitPackageVersion
+                            }), noFrameworkPackageReferenceItemGroup, mergeExisting: false);
+
+                    _transformApplicator.Execute(
```

Comment:
I don't think you can blindly add the xunit package. You should only do that if the test project already had xunit in it.